README.md

```diff
@@ -223,6 +223,38 @@ end
 See lib/redis/errors.rb for information about what exceptions are possible.
 
 
+## Timeouts
+
+The client allows you to configure connect, read, and write timeouts.
+Passing a single `timeout` option will set all three values:
```

Comment:
What units are they in? Probably seconds, but this should just be documented here.